client/app/components/conference_room/ConferenceRoomContainer.jsx

```diff
@@ -0,0 +1,70 @@
+import moment from 'moment';
+import get from 'lodash/get';
+import React from 'react';
+import EventSchema from 'proptypes/schemas/EventSchema';
+import ConferenceRoomSchema from 'proptypes/schemas/ConferenceRoomSchema';
+import { Grid, Row, Col, Jumbotron } from 'react-bootstrap';
+
+import Navbar from './layout/Navbar';
+import CurrentEvent from './event/CurrentEvent';
+import NextEvents from './event/NextEvents';
+
+import './conference_room.scss';
+
+const NO_MORE_EVENTS_TEXT = 'No more events for today';
+
+const CurrentAndNextEvents = ({ currentEvent, nextEvents, onUpdate }) => (
```

Comment:
Can we split it to multiple files? I don't know how many files are in the same directory `client/app/components/conference_room/` but maybe it would be useful to create a subdirectory and move these components there.